test/integration/tests/pcrreset.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+#;**********************************************************************;
+#
+# Copyright (c) 2016-2018, Intel Corporation
```

Comment:
I don't have a copyright on this, I didn't author/modify anything. Remove Intel.